Examples/Scripts/Python/vertex_fitting.py

```diff
@@ -32,6 +32,109 @@ class VertexFinder(enum.Enum):
+):
+    """This function steers the vertex fitting
+
+    Parameters
+    ----------
+    s: Sequencer
+        the sequencer module to which we add the Seeding steps (returned from addVertexFitting)
+    field : magnetic field
+    outputDirRoot : Path|str, path, None
+        the output folder for the Root output, None triggers no output
+    logLevel : acts.logging.Level, None
+        logging level to override setting given in `s`
+    """
+
+    def customLogLevel(custom: acts.logging.Level = acts.logging.INFO):
+        """override logging level"""
+        if logLevel is None:
+            return s.config.logLevel
+        return acts.logging.Level(max(custom.value, logLevel.value))
+
+    if int(customLogLevel()) <= int(acts.logging.DEBUG):
+        acts.examples.dump_args_calls(locals())
+
+    inputParticles = "particles_input"
+    outputVertices = "fittedVertices"
+    selectedParticles = "particles_selected"
+    trackParameters = "trackparameters"
+
+    outputTime = ""
+    if vertexFinder == VertexFinder.Truth:
+        findVertices = TruthVertexFinder(
+            level=customLogLevel(acts.logging.VERBOSE),
+            inputParticles=selectedParticles,
+            outputProtoVertices="protovertices",
+            excludeSecondaries=True,
+        )
+        s.addAlgorithm(findVertices)
+        fitVertices = VertexFitterAlgorithm(
+            level=customLogLevel(acts.logging.VERBOSE),
+            bField=field,
+            inputTrackParameters=trackParameters,
+            inputProtoVertices=findVertices.config.outputProtoVertices,
+            outputVertices=outputVertices,
+        )
+        s.addAlgorithm(fitVertices)
+
+    elif vertexFinder == VertexFinder.Iterative:
+        findVertices = IterativeVertexFinderAlgorithm(
+            level=customLogLevel(),
+            bField=field,
+            inputTrackParameters=trackParameters,
+            outputProtoVertices="protovertices",
+            outputVertices=outputVertices,
+        )
+        s.addAlgorithm(findVertices)
+    elif vertexFinder == VertexFinder.AMVF:
+        outputTime = "outputTime"
+        findVertices = AdaptiveMultiVertexFinderAlgorithm(
+            level=customLogLevel(),
+            bField=field,
+            inputTrackParameters=trackParameters,
+            outputProtoVertices="protovertices",
+            outputVertices=outputVertices,
+            outputTime=outputTime,
+        )
+
+        s.addAlgorithm(findVertices)
+    else:
+        raise RuntimeError("Invalid finder argument")
+
+    if outputDirRoot is not None:
+        outputDirRoot = Path(outputDirRoot)
+        if not outputDirRoot.exists():
+            outputDirRoot.mkdir()
+        if associatedParticles == selectedParticles:
+            warnings.warn(
+                "Using RootVertexPerformanceWriter with smeared particles is not necessarily supported. "
+                "Please get in touch with us"
+            )
```

Comment:
@paulgessinger,
This test is a bit different from the previous one (`inputTrackSummary == None with outputRoot`) in `runVertexFitting`, but I think this is what is intended. Does it look reasonable?